bundle/config/engine/engine.go

```diff
@@ -0,0 +1,56 @@
+package engine
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databricks/cli/libs/env"
+)
+
+const EnvVar = "DATABRICKS_BUNDLE_ENGINE"
+
+type EngineType string
+
+const (
+	EngineDirect    EngineType = "direct"
+	EngineTerraform EngineType = "terraform"
+	EngineNotSet    EngineType = ""
+)
+
+// Default is used for new bundles if user has not set the value
+const Default = EngineTerraform
+
+// Parse returns EngineType from string
+func Parse(engine string) (EngineType, bool) {
+	switch engine {
+	case "":
+		return EngineNotSet, true
+	case "terraform":
+		return EngineTerraform, true
+	case "direct":
+		return EngineDirect, true
+	default:
+		return EngineNotSet, false
+	}
+}
+
+// FromEnv returns engine setting from environment
+func FromEnv(ctx context.Context) (EngineType, error) {
+	value := env.Get(ctx, EnvVar)
+	engine, ok := Parse(value)
+	if !ok {
+		return EngineNotSet, fmt.Errorf("unexpected setting for %s=%#v (expected 'terraform' or 'direct' or absent/empty which means 'terraform')", EnvVar, value)
```

Comment:
This is not true looking at the acceptance tests. Absent/empty means whatever has the highest serial.